tests/ApiManifest.Tests/Helpers/ParsingHelpersTests.cs

```diff
@@ -112,5 +112,28 @@ public void ParseKeyValuePair()
             var kvPairs = ParsingHelpers.ParseKey(exampleKeyValuePair);
             Assert.Equal(3, kvPairs.Count());
         }
+
+        [Fact]
+        public async Task ParseOpenApiAsync()
+        {
+            var openApiUrl = "https://raw.githubusercontent.com/APIPatterns/Moostodon/main/spec/tsp-output/%40typespec/openapi3/openapi.yaml";
```

Comment:
I think we shouldn't have an external dependency for tests. Can we mock the fetching part instead?

Reply:
Let me look into mocking the fetch.